src/test/java/org/mockitousage/junitrule/MutableStrictJUnitTestRuleTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockitousage.junitrule;
+
+import static org.mockito.Mockito.when;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoTestRule;
+import org.mockito.quality.Strictness;
+import org.mockitousage.IMethods;
+import org.mockitoutil.JUnitResultAssert;
+
+public class MutableStrictJUnitTestRuleTest {
+
+    JUnitCore runner = new JUnitCore();
+
+    @Test public void rule_can_be_changed_to_strict() throws Throwable {
+        //when
+        Result result = runner.run(LenientByDefault.class);
+
+        //then
+        JUnitResultAssert.assertThat(result)
+                .succeeds(1)
```

Comment:
Can we make the assertion stronger and check if the right test method failed, and specific exception vs. RuntimeException. Check out the methods on JUnitResultAssert. Thanks!

Reply:
This was taken from `MutableStrictJUnitRuleTest`. Can we update these assertions in a separate PR to make them consistent?

Reply:
>This was taken from MutableStrictJUnitRuleTest. Can we update these assertions in a separate PR to make them consistent?

Absolutely!